apitools/base/py/transfer.py

```diff
@@ -632,7 +631,7 @@ def __ConfigureMultipartRequest(self, http_request):
 
     # encode the body: note that we can't use `as_string`, because
     # it plays games with `From ` lines.
-    fp = StringIO.StringIO()
+    fp = six.StringIO()  # email: cStringIO in Py2 / io.StringIO in Py3.
```

Comment:
Sadly the `email` module is a PITA. It doesn't behave consistently, so we have to use `six.StringIO()` instead of `io.StringIO()`.